beater/burrowbeat.go

```diff
@@ -84,18 +84,18 @@ func (bt *Burrowbeat) Run(b *beat.Beat) error {
 			resp, err := http.Get(endpoint)
 			if err != nil {
 				fmt.Errorf("Error during http GET: %v", err)
+			} else {
+				var burrow map[string]interface{}
```

Comment:
Actually using `interface{}` is bad pattern, but if you are not author of it, then nvm